sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -526,7 +526,7 @@ object SQLConf {
   val DEFAULT_DATA_SOURCE_NAME = buildConf("spark.sql.sources.default")
     .doc("The default data source to use in input/output.")
     .stringConf
-    .createWithDefault("parquet")
+    .createWithDefault("orc")
```

Comment:
This is a testing purpose during review.

Reply:
Can you change it back?

Reply:
Yep. It's back now, @gatorsmile .